src/styles/blocks/calendar.scss

```
@@ -0,0 +1,53 @@
+.calendar {
```

You should add vertical align to calendar
```suggestion
.calendar {
align-items: center;
```